### ISSUE TITLE
[CDCSDK] Disable the drop/truncate table on which CDC is enabled

### ISSUE THREAD

**Issue by suranjan** [yugabyte/yugabyte-db#9354]
We need to disable the drop/truncate on the table. This is to avoid unknown behavior till we provide support for drop/truncate where we will stream all the changes before a table is dropped/truncated.

**Comment by vaibhav-yb** [comment]
Truncate has been disabled for the table on which CDC is enabled, so now suppose there is a table `test` and the user executes `TRUNCATE TABLE test` he will get an error:
```
ERROR:  Invalid argument: Can't truncate a CDC enabled table : test [id=000030af000030008000000000004100]
```

Note: The id is table specific id, it might vary.

**Comment by vaibhav-yb** [comment]
The relevant details for TRUNCATE behaviour if CDC is enabled and steps on how to actually truncate the table can be found in #10010

**Comment by vaibhav-yb** [comment]
This issue is being closed in lieu of the issues #10010 and #10069
